src/main/java/frc/robot/commands/auto/LeftSixBall.java

```diff
@@ -16,7 +15,7 @@
  * Assumes someone else took our spot. Shoots 3 balls while off centered. Moves to trench to gather
  * 3 more balls. Turns and moves back to face generator and shoots collected balls.
  */
-public class LeftSixBall extends SequentialCommandGroup {
+public class LeftSixBall extends LowGearAuto {
 
   private DriveTrain drive;
```

Comment:
Remove drive train instances from subclasses. Make drive train protected in parent class.